app/javascript/components/Tweetes.jsx

```diff
@@ -19,9 +19,6 @@ class Tweetes extends Component {
             })
             .then(response => this.setState({ tweetes: response }))
             .catch(() => this.props.history.push("/"));
-            // fetch(url)
```

Comment:
Very good! Commented out code leads to noise. As more people contribute to the same code base, commented out code tends to persist because nobody knows if the commented out code is serving a purpose. Always try to remove commented out code if you don't need it anymore.